keyboards/ergodox_ez/keymaps/bepo_tm_style/readme.md

```diff
@@ -13,10 +15,14 @@ Arduino environment and Teensy loaded in the default path, you can do:
 
 ```shell
 PATH=/cygdrive/c/Program\ Files\ \(x86\)/Arduino/hardware/tools/avr/bin:$PATH
+make git-submodule
 make ergodox_ez:bepo_tm_style:all
 ```
+The last command will fail when trying to copy the firmware file. But, at that
+stage, the file is already ready. The only issue is that it’s size is not
+checked.
```

Comment:
This whole section (lines 11 to 23) should not be needed. We recommend using MSYS2 and the CLI to compile on Windows:
https://docs.qmk.fm/#/newbs_getting_started?id=windows